strings/tcg/set_one.json

```diff
@@ -464,70 +464,70 @@
 			"id": "ert_command",
 			"name": "NT P.A.V. Suit (Command)",
 			"desc": "Issued to members of Emergency Response Teams, the P.A.V. Suit gives superior protection from any threat the galaxy can throw at it. This particular model is outfitted with a sidearm holster and a sleek blue finish.",
-			"rules": "While equipped, give the equipped unit {$Squad Tactics} and {$First Strike}.",
+			"rules": "While equipped, give the equipped unit {$First Strike}.",
 			"icon_state": "ert_command",
-			"power": "2",
-			"resolve": "2",
+			"power": "0",
+			"resolve": "0",
 			"faction": "Command",
 			"summoncost": "2",
 			"cardtype": "Equipment",
-			"cardsubtype": "Armour",
+			"cardsubtype": "Command ERT",
 			"rarity": "rare"
 		},
 		{
 			"id": "ert_engi",
 			"name": "NT P.A.V. Suit (Engineering)",
 			"desc": "Issued to members of Emergency Response Teams, the P.A.V. Suit gives superior protection from any threat the galaxy can throw at it. This particular model is outfitted with a welding screen and a flashy yellow finish.",
-			"rules": "While equipped, give the equipped unit {$Squad Tactics}.",
+			"rules": "While equipped, give the equipped unit {$Squad Tactics}: Gain +1/+1 until the end of the turn.",
 			"icon_state": "ert_engi",
 			"power": "1",
 			"resolve": "1",
 			"faction": "Engineering",
 			"summoncost": "1",
 			"cardtype": "Equipment",
-			"cardsubtype": "Armour",
+			"cardsubtype": "ERT Atmosian",
 			"rarity": "uncommon"
 		},
 		{
 			"id": "ert_med",
 			"name": "NT P.A.V. Suit (Medical)",
 			"desc": "Issued to members of Emergency Response Teams, the P.A.V. Suit gives superior protection from any threat the galaxy can throw at it. This particular model is outfitted with a sterile coating and a calming white finish.",
-			"rules": "While equipped, give the equipped unit {$Squad Tactics}.",
+			"rules": "Whenever the equipped unit deals damage it heals you for that amount.",
```

Comment:
Clarify "you" here, does this heal the equipped creature or the player's life total?

Reply:
Clarified, the player's